pandas/_libs/window/meson.build

```diff
@@ -0,0 +1,18 @@
+py.extension_module(
+    'aggregations',
+    ['aggregations.pyx'],
+    include_directories: [inc_np, '../src'],
```

Comment:
Can this be converted to an absolute include path? Don't think this is relative in setup.py today

Reply:
It's possible with ``meson.source_root``, but it would make it a lot more verbose. 

Personally, I would keep it as is. I don't think the directory structure of _libs has changed in a while.